### ISSUE TITLE
Tooltip Opacity

### ISSUE THREAD

**Issue by geoffgpe** [grafana/grafana#79608]
**Why is this needed**:

Tooltip Opacity is needed as it is useful to see what is under the Tooltip. [1]

**What would you like to be added**:

I would like the ability to select tooltip opacity such as a number between 0 to 100. 

Without the ability to specify the opacity of the tooltip, it is not possible to see what is under the tooltip and much of the visual information and impact of the graph can be lost.

**Who is this feature for?**

Any user who produces a graph and loses some kind of information provided by the graph due the tooltip obscuring the details. It will also save the user time trying to adjust the graph to size and move the tooltip to get the most out of an awkward and unnecessary loss of visual information.

Note:

[1] Apologies if this feature is already available. An internet search produces minimal information and it is not in the obvious places where it could be available.

<img width="1043" alt="Screenshot 2023-12-17 at 09 37 26" src="https://github.com/grafana/grafana/assets/100133517/7833c82c-5c62-4a9f-abac-53c36217a453">

**Comment by leeoniya** [comment]
> [1] Apologies if this feature is already available. An internet search produces minimal information and it is not in the obvious places where it could be available.

no, it does not currently exist. as far as i know this is the first ask for it.

**Comment by geoffgpe** [comment]
Hi Leon,

Thanks for the quick response and confirmation that the feature is not available.

It does not surprise me that you confirm that it has not been requested due to lack of search responses.

But it does surprise me that it has not been requested before. I face the issue every time I want to share a screenshot of a graph (multiple times per day). I must be a lone user 😊

Hope it is something implementable without too much effort.

Thanks again.

Regards, Geoff


[Image]
Geoff Eldridge
Principal Advisor

M. +61 435 770 322
E. ***@***.******@***.***>
www.globalpowerenergy.com.au<http://www.globalpowerenergy.com.au/>
[Image]



From: Leon Sorokin ***@***.***>
Date: Sunday, 17 December 2023 at 10:44
To: grafana/grafana ***@***.***>
Cc: Geoff Eldridge ***@***.***>, Author ***@***.***>
Subject: Re: [grafana/grafana] Tooltip Opacity (Issue #79608)

[1] Apologies if this feature is already available. An internet search produces minimal information and it is not in the obvious places where it could be available.

no, it does not currently exist. as far as i know this is the first ask for it.

**Comment by leeoniya** [comment]
> Hope it is something implementable without too much effort.

the effort is low, but we also have to balance how many options we show to all our users. it's not so much an engineering challenge but one of keeping the sheer quantity of options we expose to a sane level.

let's see how many upvotes this gets in the coming months and we'll circle back.

**Comment by geoffgpe** [comment]
Hi Leon,

Thanks again for keeping me informed. Really appreciate it.

Totally understand the process you are using. Hope the request gets some traction, but it appears I am using Grafana in a way that most don’t – i.e. using screenshots of Grafana Dashboards to convey a message such as LinkedIn post, etc ..

https://www.linkedin.com/feed/update/urn:li:activity:7141938571535392768/

For me in software development I find some beauty in the process when a feature finds a natural home amongst the other ui features. I am not sure a feature such as requested adds too much complexity or clutter to the UI when it sits in the right place particularly when it’s exceptional useful and functional.

Thanks for listening and responding so promptly.

Regards, Geoff

From: Leon Sorokin ***@***.***>
Date: Sunday, 17 December 2023 at 12:08
To: grafana/grafana ***@***.***>
Cc: Geoff Eldridge ***@***.***>, Author ***@***.***>
Subject: Re: [grafana/grafana] Tooltip Opacity (Issue #79608)

Hope it is something implementable without too much effort.

the effort is low, but we also have to balance how many options we show to all our users. it's not so much an engineering challenge but one of keeping the sheer quantity of options we expose to a sane level.

let's see how many upvotes this gets in the coming months and we'll circle back.
